depends/hosts/darwin.mk

```diff
@@ -0,0 +1,17 @@
+OSX_MIN_VERSION=10.7
+OSX_SDK_VERSION=10.9
```

Comment:
As above, xcode is not used. We control the SDK used for builds, the desired version is set here.